sklearn/feature_selection/tests/test_feature_select.py

```diff
@@ -968,3 +968,15 @@ def test_mutual_info_regression():
     gtruth = np.zeros(10)
     gtruth[:2] = 1
     assert_array_equal(support, gtruth)
+
+
+@pytest.mark.parametrize("dtype_in", [np.float32, np.float64])
+def test_empty_support_dtype(dtype_in):
```

Comment:
There is already a test here:

https://github.com/scikit-learn/scikit-learn/blob/6440856fbb0e1c0a048316befbf6df4e0a5765c1/sklearn/feature_selection/tests/test_feature_select.py#L445-L449

We can expand on that test by using parametrize on "dtype_in" and then checking the dtype at the end.

Reply:
Oh nice, thanks for showing me that one. Adjusted!